src/room/GroupCallView.tsx

```diff
@@ -141,14 +141,14 @@ export function GroupCallView({
         PosthogAnalytics.instance.eventCallStarted.track(groupCall.groupCallId);
 
         await Promise.all([
-          widget.api.setAlwaysOnScreen(true),
-          widget.api.transport.reply(ev.detail, {}),
+          widget?.api.setAlwaysOnScreen(true),
+          widget?.api.transport.reply(ev.detail, {}),
```

Comment:
```suggestion
          widget!.api.setAlwaysOnScreen(true),
          widget!.api.transport.reply(ev.detail, {}),
```

due to the `if (widget)` above

Reply:
Just to double-check: we guarantee that the `widget` cannot be modified otherwise, right?

The reason I'm asking is that at this place the `widget` is used from the callback that is defined inside the `if`, not from the `if` block by itself, hence the warning from a transpiler, i.e. by the point the callback is called, the widget may become `null` again. The `on()` action that subscribes to the event does not trigger this warning since it's called directly inside an `if` (i.e. guaranteed to be non-null).

Reply:
Correct, `widget` can't be modified because it's a `const`.

I think the reason that TypeScript doesn't cast it to a non-null value here is because you're supposed to be able to replace a declaration like `export const widget` with `export let widget` without that being a breaking change in your module's API, and so the type checker has to pretend that `widget` could change to `null` by the time the function is invoked.

Reply:
I see, thanks for explaining! Fixed.